sql/core/src/test/resources/sql-tests/inputs/datetime.sql

```diff
@@ -1,5 +1,10 @@
 -- date time functions
 
+-- [SPARK-31710] TIMESTAMP_SECONDS, TIMESTAMP_MILLISECONDS and TIMESTAMP_MICROSECONDS to timestamp transfer
+select TIMESTAMP_SECONDS(1230219000),TIMESTAMP_SECONDS(-1230219000),TIMESTAMP_SECONDS(null);
```

Comment:
Is it possible to test an out-of-range case, i.e., where the *1000 or *1000000 overflows, and to show that it returns an error?

Reply:
Maybe in `org.apache.spark.sql.SQLQuerySuite` we add the overflows added ?

Reply:
The SQL test suite  can also show exceptions, so let's keep it there as well.

Reply:
+1

Reply:
Ok, I will add exception check.